src/api/tests/test_mdflow_adapter.py

```
@@ -0,0 +1,173 @@
+        block = BlockDTO(
+            bid="test10", block_content=content, variable_bids=[], resource_bids=[]
+        )
+
+        result = convert_block_to_mdflow(block, {})
+        expected = "测试中文字符：你好世界！"
+        assert result == expected
+        mock_print.assert_called_once_with(expected)
+
+    @patch("flaskr.service.shifu.block_to_mdflow_adapter.print")
+    @patch("flaskr.service.shifu.block_to_mdflow_adapter.raise_error")
+    def test_convert_block_invalid_type(self, mock_raise_error, mock_print):
```

Ruff (0.12.2)

Unused method argument: `mock_print`

(ARG002)